### PR TITLE
deleted tuinity

### PR DESCRIPTION
**Pull request by cubelius**
because this .jar is no more supported

### CODE REVIEW

File: common-pitfalls-and-best-practices.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ There are two types of people - those who make backups, and those who will start
...
+Bukkit and Spigot are basically in maintenance mode. They update anytime there's a new version and if a critical exploit is found, but don't add any performance updates. This means any performance issues you may experience on those softwares will never be improved over time. To avoid that, upgrade to [Paper](https://papermc.io/downloads), [Purpur](https://purpur.pl3x.net/downloads) or another stable and good fork. Bukkit/Spigot plugins will work just as well (maybe even better) with the server software listed. If they don't, then it's safe to assume that the plugin dev is either doing things that they shouldn't or did a negligent job creating their plugin. They also add optimization patches like a chunk loading system that can take advantage of multiple cpu threads or a setting that allows the server to tick less chunks than it actually sends to the player. See the [main optimization guide](https://github.com/YouHaveTrouble/minecraft-optimization) for more details.
```
Comment (YouHaveTrouble) [inline common-pitfalls-and-best-practices.md:12]: either define "another stable and good fork" or don't add it at all.

**Comment by YouHaveTrouble** [comment]
Patina is yatopia with a couple patches on top, can't recommend that with good conscience

**Comment by cubelius** [comment]
although it is a .jar from Foss, which is quite productive. good, so there is no need to mention other forks for now.

**Comment by cubelius** [comment]
at least I did'nt notice such dangerous patches in patinamc, which were in yatopia

**Pull request merged**